docs/first-order/current_source_density.ipynb

```diff
@@ -0,0 +1,992 @@
+{
```

Comment:
<p>Same, I suggest adding a imshow of only one Stimulus response (not all 150 of them) to allow sanity checking there is a response, although noisy.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>